sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/options/BlobChangeLeaseOptions.java

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.blob.options;
+
+import com.azure.core.annotation.Fluent;
+import com.azure.storage.blob.models.BlobLeaseRequestConditions;
+import com.azure.storage.common.implementation.StorageImplUtils;
+
+/**
+ * Extended options that may be passed when changing a lease to a blob or container.
+ */
+@Fluent
+public class BlobChangeLeaseOptions {
+
+    private final String proposedId;
+    private BlobLeaseRequestConditions requestConditions;
+
+    /**
+     * @param proposedId A new lease ID in a valid GUID format.
+     */
+    public BlobChangeLeaseOptions(String proposedId) {
+        StorageImplUtils.assertNotNull("proposedId", proposedId);
```

Comment:
Is this a valid check here?

Reply:
I think so. We probably want to be doing this on all the constructor values.